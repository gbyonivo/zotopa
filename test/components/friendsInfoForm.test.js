import React from 'react';
import { create } from 'react-test-renderer';
import FriendsInfoForm from '../../src/components/friendInfoForm';

describe('FriendsInfoForm', () => {
  it('should render correctly', () => {
    const actual = create(<FriendsInfoForm
      name = "barry"
      email = "milner@gmail.com"
      onChange={() => {}}
      emailError="email error"
      nameError="name error"
    />);
    expect(actual).toMatchSnapshot();
  });
});