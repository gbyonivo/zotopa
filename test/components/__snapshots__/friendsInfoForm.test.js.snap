// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendsInfoForm should render correctly 1`] = `
<div>
  <div
    className="textInput"
  >
    <label
      htmlFor="name"
    >
      Name
    </label>
    <input
      name="name"
      onChange={[Function]}
      placeholder="Ngufan Ivo"
      type="text"
      value="barry"
    />
    <span
      className="error"
    >
      name error
    </span>
  </div>
  <div
    className="textInput"
  >
    <label
      htmlFor="email"
    >
      E-mail
    </label>
    <input
      name="email"
      onChange={[Function]}
      placeholder="orkumaivo@gmail.com"
      type="text"
      value="milner@gmail.com"
    />
    <span
      className="error"
    >
      email error
    </span>
  </div>
</div>
`;
